client/src/app/dashboard/dashboard-overview/dashboard-overview.component.html

```diff
@@ -19,18 +19,175 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-md-3 col-sm-4">
-    <app-dashboard-card route="crawler" componentType={{crawler}} title="Crawler" img="../assets/images/crawler.png"></app-dashboard-card>
+      <app-dashboard-card route="crawler" componentType={{crawler}} title="Crawler" img="../assets/images/crawler.png" collapse_target="#demo1" numberOfNotification="3"></app-dashboard-card>
```

Comment:
please change the collapse targets ids to something meaningful. 
create a variable for the numberOfNotifications and use that one depending on the component type